src/internal/connector/sharepoint/restore.go

```diff
@@ -85,3 +88,59 @@ func createRestoreFolders(ctx context.Context, service graph.Servicer, siteID st
 
 	return onedrive.CreateRestoreFolders(ctx, service, *mainDrive.GetId(), restoreFolders)
 }
+
+// restoreItem utility function restores a List to the siteID. The name is changed to to Corso_Restore_{timeStame}_name
+// API Reference: https://learn.microsoft.com/en-us/graph/api/list-create?view=graph-rest-1.0&tabs=http
+// Restored List can be verified within the Site contents
+func restoreItem(
```

Comment:
Since this is only used to restore lists, and not all sharepoint items, I think it's appropriate to namespace here.
```suggestion
func restoreListItem(
```

Reply:
Change within [f5e2712](https://github.com/alcionai/corso/pull/1961/commits/f5e27121870712e3aa7ca6f363182dd3c53c75be)